packages/spark/sodasql/dialects/spark_dialect.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, parser: Parser):
             self.port = parser.get_int_optional('port', '10000')
             self.username = parser.get_credential('username')
             self.password = parser.get_credential('password')
-            self.database = parser.get_str_optional('database', 'default')
+            self.database = parser.get_str_optional('database')
```

Comment:
We should test how this works out for the `hive` connector. As suggested in [this PR](https://github.com/sodadata/soda-sql/pull/481), it would be nice to have a parser per type of connection, so that we can have different requirements on the user input.